jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java

```diff
@@ -277,6 +278,7 @@ public PreemptStatus tryPreempt(Thread t) {
 	}
 
 	/* Continuation Native APIs */
+	@JvmtiMountTransition
```

Comment:
I have added the annotation for `yieldImpl` since it has similar traits as `enterImpl`. Currently, adding the annotation for `yieldImpl` has nil impact because `yieldImpl` always happens between `notifyJvmtiUnmountBegin`and `notifyJvmtiUnmountEnd` for a virtual thread. JVMTI operations cannot happen between the two `notifyJvmti*` methods due to synchronization.